src/chartkit/components/Pie.js

```diff
@@ -41,10 +64,17 @@ class Pie extends Component {
     }));
 
     return (
-     <PieWrapper style={this.props.style}>
-       {this.props.title ? <PieTitle>{this.props.title}</PieTitle> : null }
-       <ResponsivePie {...this.props} data={colorData} colorBy={data => data.color} />
-     </PieWrapper>
+      <PieWrapper style={this.props.style}>
+        {this.props.title ? <PieTitle>{this.props.title}</PieTitle> : null}
```

Comment:
Thoughts on pulling `title` and `style` out above it it with `const {style, tittle} = this.props` to clean it up a bit?

Reply:
@nyanofthemoon

Reply:
I'm not opposed to neither for class components. Though I personally think functional components remove these kind of noise, but it's mainly a personal preference :P

Reply:
I'm not opposed to neither for class components. Though I personally think functional components remove these kind of noise, but it's mainly a personal preference :P

Reply:
Not opposed, just wanted to do the cleanup after plugging in the data provider; there might be additional things to cleanup.